src/renderer/store/modules/sponsorblock.js

```diff
@@ -0,0 +1,31 @@
+import $ from 'jquery'
+
+const state = {}
+const getters = {}
+
+const actions = {
+  sponsorBlockSkipSegments ({ rootState }, { videoId, categories }) {
+    return new Promise((resolve, reject) => {
+      const requestUrl = `${rootState.settings.sponsorBlockUrl}skipSegments?videoID=${videoId}&categories=${JSON.stringify(categories)}`
```

Comment:
Nice job! By the way, if it is possible to do a sha 256 hash of the videoID client-side, you can use the [hash prefix API](https://github.com/ajayyy/SponsorBlock/wiki/API-Docs#get-apiskipsegmentssha256hashprefix) to get this in a more privacy friendly way.

Reply:
implemented in https://github.com/FreeTubeApp/FreeTube/pull/1130/commits/09b9b4522846d3a3b65faf8e1c86e55d2776d590